plugins/catalog-backend/src/next/Stitcher.ts

```diff
@@ -149,9 +149,14 @@ export class Stitcher {
           };
         }
         if (errors) {
-          const parsedErrors = JSON.parse(errors);
+          const parsedErrors = JSON.parse(errors) as SerializedError[];
           if (Array.isArray(parsedErrors) && parsedErrors.length) {
-            processingStatus.errors = parsedErrors;
+            statusItems = parsedErrors.map(e => ({
+              type: ENTITY_STATUS_CATALOG_PROCESSING_TYPE,
+              level: 'error',
+              message: e.message,
```

Comment:
Would kinda prefer `e.toString()` here tbh, since we're already kinda somewhere in the middle between nice presentation and more techy messaging. Main reason is that `failed to X, NotFoundError: could not find Y` is a bit weird compared to `NotAllowedError: failed to X, NotFoundError: could not find Y`

Reply:
Sure!